pycnn.py

```diff
@@ -71,6 +113,20 @@ def validate(self, input_location):
     # tempA: feedback template, tempB: control template
     def imageprocessing(self, inputlocation, outputlocation,
                         tempA, tempB, initialcondition, Ib, t):
+        """Process the image with the input arguments.
+
+        Args:
+            inputlocation (str): The string path for the input image.
+            outputlocation (str): The string path for the output processed
+                image.
+            tempA (:obj:`list` of :obj:`list`of :obj:`float`): Feedback
+                template.
+            tempB (:obj:`list` of :obj:`list`of :obj:`float`): Control
+                template.
+            initialcondition (float): The initial template state.
```

Comment:
initialcondition isn't related to template, you can simply refer it as "The initial state."